src/data/engineering.js

```diff
@@ -254,6 +254,26 @@ const ILL_ADVISED_ACTIONS = {
 			power: 11
 		}
 	},
+	engSuperMatter: {
+		time: 5,
+		actionName: "THROW COIN AT",
+		name: "Supermatter Crystal",
+		failure: {
+			chance : 0.1,
+			damage: 1000,
+			message: "Dusted"
+		},
+		items: {
+			id: "power",
+			count: 100

```

Comment:
I'm not sure I'm a super big fan of this being the highest power/s in the game. You could realistically go nude and actively play to get much better power gen than bananium, oil slimes, or any other good power generation method that relies on other skills.

I'm OK with active play rewarding _slightly_ higher power/s, but right now this is giving ~18 power / s, while oil slimes (the current best method in the game) give ~3 power/s

So I'd say either
- Bump up the time significantly:
- Lower the power gain
- Put some hefty -xp on this (making it a better, but more risky singulo)
- Make it a lot of coins instead

Reply:
You bring up a good point.

I will do a little math tomorrow to estimate which of these I should do.